rotkehlchen/tests/unit/decoders/test_convex.py

```diff
@@ -99,6 +106,17 @@ def test_booster_deposit(database, ethereum_inquirer, ethereum_accounts):
         database=database,
         tx_hash=evmhash,
     )
+    mocked_notifier = database.msg_aggregator.rotki_notifier
+    mocked_notifier.pop_message()  # detected stETH token
```

Comment:
test for those. Why just add comments? If this is what it's supposed to be then you should test for them.